app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffectHandler.kt

```diff
@@ -73,6 +76,9 @@ class ChangeLanguageEffectHandler @AssistedInject constructor(
                 .subscribeOn(scheduler)
                 .toSingleDefault(newLanguage)
           }
+          .doOnNext {
+            dataSync.fireAndForgetSync(DAILY)
```

Comment:
I feel that deciding whether to trigger a sync or not should be a business logic decision and handling an effect should follow the `Single Responsibility Principle`.

What do you think about converting this into a separate effect, say `TriggerSync`, and then have `ChangeLanguageUpdate` emit that effect when it receives the `CurrentLanguageChangedEvent`?